page-objects/pages/teamsPages/TMSTeamListPage.js

```diff
@@ -131,14 +161,40 @@ async function clickMemberIconInTeam(teamName) {
 	let pupilIcon = await teamWrapper.$(memberBtn);
 	await elementHelpers.clickAndWait(pupilIcon);
 }
+
+async function clickOnChosenTeam(teamName) {
+	const element = await getWrapperOfTeam(teamName);
+	await elementHelpers.clickAndWait(element);
+}
+
+async function isTeamVisible(teamName, expectedValue) {
+	const defaultString = `Team with name: ${teamName}`;
+
+	const msg = expectedValue
+		? `${defaultString} should be visible on the list`
+		: `${defaultString} should not be visible on the list`;
+
+	const resultMsg = 'Actual list of teams: ' + (await getListOfTeamTitles());
+	const isTeam = await isTeamOnList(teamName);
+	expect(isTeam, msg + resultMsg).to.equal(expectedValue);
+
+}
+
+async function isTeamOnList(teamName) {
```

Comment:
Separate method not needed. Use everywhere isTeamVisible

Reply:
i removed another one this method is used in isTeamVisible